jwst/ramp_fitting/ramp_fit.py

```diff
@@ -657,13 +827,11 @@ def gls_ramp_fit(model,
 
     # Create new model...
     if n_int > 1:
-        new_model = datamodels.ImageModel(data=slopes.astype(np.float32),
-                                      dq=final_pixeldq,
-                                      err=gls_err.astype(np.float32))
+        new_model = datamodels.ImageModel(data=med_slopes.astype(np.float32),
```

Comment:
Assuming that this is the data model that will end up as the `_rate.fits` product, where are the `med_slopes` values coming from? I thought it looked like earlier that these were the slopes computed from the simple process of taking the first-differences of all the groups. But what we really want here are the slopes computed from the "real" fitting process, and for n_int>1 the result of taking the average of those fits over all integrations.